### ISSUE TITLE
Synopsis is missing the -r option

### ISSUE THREAD

**Issue by autarch** [cpanel/Test2-Harness-Renderer-JUnit#2]
As it stands, the example in the synopsis will simply run with the regular renderer, not this one.

**Comment by toddr** [comment]
Crap don’t tell me you’re using this thing!

I’ll give you our command in a few mins

**Comment by autarch** [comment]
Well, I'm _trying_ to use it but #3 is causing me some real problems.

**Comment by toddr** [comment]
I'm at a conference so I can't look close for 48 hours. We're doing something like this.

```
JUNIT_TEST_FILE="/tmp/foo.xml" ALLOW_PASSING_TODOS=1 yath test -j18 --retry=1 --retry-job-count=1 --renderer=Formatter --renderer=JUnit t/*.t
```

**Comment by toddr** [comment]
This was fixed with c84493890327c01f4012c05555934db8faeb6d00
